data_processing/common/Container.py

```diff
@@ -71,24 +74,42 @@ def __init__(self, params):
         #self.logger = init_logger("common-container.log", "Container [empty]")
         self.logger = logging.getLogger(__name__)
 
-        self.params=params
+        if isinstance(params, ConfigSet):
+            params=params.get_config_set("CONTAINER_CFG")
 
         # Connect to graph DB
         self.logger.info ("Connecting to: %s", params['GRAPH_URI'])
         self._conn = Neo4jConnection(uri=params['GRAPH_URI'], user=params['GRAPH_USER'], pwd=params['GRAPH_PASSWORD'])
         self.logger.info ("Connection test: %s", self._conn.test_connection())
+
+        self.logger.info ("Connecting to: %s", params['MINIO_URI'])
+        self._client = Minio(params['MINIO_URI'], access_key=params['MINIO_USER'], secret_key=params['MINIO_PASSWORD'], secure=False)
+        try:
+            for bucket in self._client.list_buckets():
+                self.logger.info("Found bucket %s", bucket.name )
+            self.logger.info("object_store_enabled=True")
+            params['object_store_enabled'] = True
+        except:
+            self.logger.warning("object_store_enabled=False")
+            params['object_store_enabled'] = False
+
```

Comment:
i would suggest to simply have `object_store_enabled` as a user input, with some default, and set up a client only when the param is true.